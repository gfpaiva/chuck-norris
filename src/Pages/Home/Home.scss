@import '../../Scss/root';

.home {
  &__welcome {
    &-title {
      @include shadow();
      margin: -30px auto 3rem;
      transform: skewY(-10deg);
      max-width: 500px;
    }
  }
}
