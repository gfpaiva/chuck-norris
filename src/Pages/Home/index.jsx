import React from 'react';

import CategoryGrid from '../../Components/CategoryGid';

import chuck from './chuck-norris-welcome.png';

import './Home.scss';

export default function Home() {
  return (
    <div className="home">
      <div className="home__welcome text-center">
        <img className="home__welcome-image full-width" src={chuck} alt="Chuck Norris" title="Chuck Norris" />

        <h1 className="home__welcome-title bg-secondary py-1">
          <span>Chuck Facts</span>
        </h1>

        <p className="home__welcome-text">Selecione uma das categorias abaixo para &quot;Chuck Facts&quot;</p>
      </div>
      <CategoryGrid />
    </div>
  );
}
