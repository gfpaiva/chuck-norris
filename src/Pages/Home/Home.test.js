import React from 'react';
import { shallow } from 'enzyme';

import Home from './index';

describe('<Home />', () => {
  it('should mount properly', () => {

    const wrapper = shallow(<Home />);

    expect(wrapper).toMatchSnapshot();
  })
});
