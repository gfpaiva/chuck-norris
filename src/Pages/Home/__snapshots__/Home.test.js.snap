// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should mount properly 1`] = `
<div
  className="home"
>
  <div
    className="home__welcome text-center"
  >
    <img
      alt="Chuck Norris"
      className="home__welcome-image full-width"
      src="test-file-stub"
      title="Chuck Norris"
    />
    <h1
      className="home__welcome-title bg-secondary py-1"
    >
      <span>
        Chuck Facts
      </span>
    </h1>
    <p
      className="home__welcome-text"
    >
      Selecione uma das categorias abaixo para "Chuck Facts"
    </p>
  </div>
  <Connect(CategoryGrid) />
</div>
`;
